src/config-site/windows.cmake

```diff
@@ -232,3 +232,13 @@ VISIT_OPTION_DEFAULT(VISIT_XDMF_LIBDEP
     ${VISIT_VTK_DIR}/lib vtklibxml2-${VTK_MAJOR_VERSION}.${VTK_MINOR_VERSION}
     TYPE STRING)
 
+
+##
+## ADIOS2
+##
+SETUP_APP_VERSION(ADIOS2 2.5.0)
+VISIT_OPTION_DEFAULT(VISIT_ADIOS2_DIR ${VISITHOME}/adios2-ser/${ADIOS2_VERSION})
+## (configured w/ mpi compiler wrapper)
+if(VISIT_PARALLEL)
```

Comment:
I notice you made this conditional on VISIT_PARALLEL, none of the other config site files for ADIOS2 do that and they seem to work fine. More of a question if this is necessary, than anything else.

Reply:
I believe most config-site files also turn on VISIT_PARALLEL by default. If VISIT_PARALLEL isn't turned on then the mpi-enabled libraries cause a build failure.

Reply:
Yes, all the systems I built ADIOS2 on were parallel builds. Good to know.